tests/test_vxlan_vtep_vni.rb

```diff
@@ -153,6 +155,8 @@ def test_multicast_group
   end
 
   def test_peer_list
+    skip('Platform does not support peer-list') unless
+    node.cmd_ref.supports?('vxlan_vtep_vni', 'peer_list')
     vni = VxlanVtepVni.new('nve1', '6000')
     if validate_property_excluded?('vxlan_vtep_vni', 'ingress_replication')
```

Comment:
I believe this is a copy paste error?  Argument should be `peer_list` instead of `ingress_replication`